### PR TITLE
Update roadmap.html

### PR DESCRIPTION
**Pull request by saimaghafoor**
Content tweaks to roadmap page

### CODE REVIEW

**Comment by saimaghafoor** [comment]
@karlchillmaid I've added a bullet under the heading 'Things we're working on' . It's  on line 36 'making it easier for services to go-live'. Adapted from the roadmap content doc. Please check it's accuracy!

File: app/templates/views/roadmap.html [hunk 1]
```diff
@@ -29,69 +29,42 @@ <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working
...
-    <li>Let services choose their own two-factor authentication method</li>
-    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_receive_text_messages') }}">receive text messages</a> without having to contact the Notify team first</li>
-    <li>Make it easier to find information and guidance on the Notify website</li>
-    <li>Design reusable content patterns for text messages</li>
-    <li>Update the version of the <a class="govuk-link govuk-link--no-visited-state" href="https://design-system.service.gov.uk/">GOV.UK Design System</a> that we use</li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrating our database and applications from GOV.UK PaaS</a> to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
+    <li>Add multilingual letter templates and attachments to letters so services can provide information in several languages</li>
+    <li>Improve our email functionality</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:35]: `Improve our email functionality` feels a bit vague - what do we mean by this?

File: app/templates/views/roadmap.html [hunk 2]
```diff
@@ -29,69 +29,42 @@ <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working
...
-    <li>Let services choose their own two-factor authentication method</li>
-    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_receive_text_messages') }}">receive text messages</a> without having to contact the Notify team first</li>
-    <li>Make it easier to find information and guidance on the Notify website</li>
-    <li>Design reusable content patterns for text messages</li>
-    <li>Update the version of the <a class="govuk-link govuk-link--no-visited-state" href="https://design-system.service.gov.uk/">GOV.UK Design System</a> that we use</li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrating our database and applications from GOV.UK PaaS</a> to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
+    <li>Add multilingual letter templates and attachments to letters so services can provide information in several languages</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:34]: Can we be consistent about whether our verbs end in `-ing`?

My preference is whatever is easiest to read. In the past we’ve gone with ‘migrate’, ‘design’, ‘make’, ‘update’ – but it’s your call. 🙂

File: app/templates/views/roadmap.html [hunk 3]
```diff
@@ -29,69 +29,42 @@ <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working
...
+    <li>Improve our email functionality</li>
+    <li>Making it easier for services to go-live</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:36]: We tend to talk about a ‘request to go live’ rather than just ‘go live’ or ‘go-live’ in isolation.

I think we could be better about explaining trial mode and live in general – but that’s a problem for another day.

For now I might suggest something like:

```suggestion
    <li>Make it easier for services to send a request to go live</li>
```

Or

```suggestion
    <li>Make it easier for services to remove trial mode restrictions</li>
```

Or a combination of the 2. 

I haven’t been super close to this work, though, so perhaps this doesn’t adequately capture the actual changes we’ve made?

File: app/templates/views/roadmap.html [hunk 4]
```diff
@@ -29,69 +29,42 @@ <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working
...
+    <li>Design and test reusable content patterns for text messages</li>
+    <li>Start sending our letters through API inegration with our letter provider</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:38]: This one doesn’t communicate the user benefit. That’s ok if there isn’t a tangible one. It’s fine to talk about backend improvements we’re making. However, this one makes it sound a bit like ‘start sending letters’ is the thing – which could be confusing. Perhaps we should frontload the ‘Integrate with our letter provider’s API’ aspect?

File: app/templates/views/roadmap.html [hunk 5]
```diff
@@ -29,69 +29,42 @@ <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working
...
     <li>Improve our front-end code to meet our <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.accessibility_statement')}}">targets for accessibility</a></li>
 
   </ul>
 
   <h2 class="heading-medium" id="things-we-will-do-later">Things we’ll do later</h2>
 
-  <h3 class="heading-small" id="january-to-march">January to March 2023</h3>
+  <h3 class="heading-small" id="april-to-june">Spring/Summer 2023</h3>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:45]: I’m not sure about the backslash.

‘Spring and summer’?
‘April to September’ (or whatever the actual months we’re talking about are)?

File: app/templates/views/roadmap.html (hunk 2 above)
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:34]: I find ‘Add multilingual letter templates and attachments to letters so services can provide information in several languages’ a bit hard to parse.

Could we split it in 2 and say something like:

```suggestion
    <li>Let services format their letters in Welsh</li>
    <li>Let services attach extra pages, like forms, to letters</li>
```

File: app/templates/views/roadmap.html [hunk 6]
```diff
@@ -29,69 +29,42 @@ <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working
...
-    <li>Make it quicker and easier to turn off <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.trial_mode')}}">trial mode restrictions</a></li>
-    <li>Let services choose their own two-factor authentication method</li>
-    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_receive_text_messages') }}">receive text messages</a> without having to contact the Notify team first</li>
-    <li>Make it easier to find information and guidance on the Notify website</li>
-    <li>Design reusable content patterns for text messages</li>
-    <li>Update the version of the <a class="govuk-link govuk-link--no-visited-state" href="https://design-system.service.gov.uk/">GOV.UK Design System</a> that we use</li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrating our database and applications from GOV.UK PaaS</a> to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
```
Comment (quis) [inline app/templates/views/roadmap.html:33]: _database_ replicates the point above
```suggestion
    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrating our applications from GOV.UK PaaS</a> to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
```

File: app/templates/views/roadmap.html [hunk 7]
```diff
@@ -15,83 +15,60 @@ <h1 class="heading-large">Roadmap</h1>
...
+    <li>Let services create letters in the Welsh language to comply with the Welsh Language Act 1993</li>
+    <li>Add attachments to letters</li>
+    <li>Improve our email functionality so users can add images, QR codes and have more formatting options</li>
+    <li>Make it easier for services to request to go-live</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:36]: Not sure this needs a hyphen

```suggestion
    <li>Make it easier for services to request to go live</li>
```

File: app/templates/views/roadmap.html [hunk 8]
```diff
@@ -15,83 +15,60 @@ <h1 class="heading-large">Roadmap</h1>
...
+    <li>Make it easier for users to join a live service</li>
+    <li>Design and test reusable content patterns for text messages</li>
+    <li>Integrate with our letter provider's API to improve the letter sending process</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:39]:
```suggestion
    <li>Integrate with our letter provider’s API to improve the letter sending process</li>
```

File: app/templates/views/roadmap.html (hunk 6 above)
Comment (karlchillmaid) [reply app/templates/views/roadmap.html:32]: Taking Chris’s suggestion and making it consistent with the other verbs in the list.

```suggestion
    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrate our applications from GOV.UK PaaS</a> to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
```

File: app/templates/views/roadmap.html [hunk 9]
```diff
@@ -15,83 +15,60 @@ <h1 class="heading-large">Roadmap</h1>
...
+  <h3 class="heading-small" id="april-to-june">April to August 2023</h3>
 
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>Improve formatting options for letters</li>
-    <li>Provide more detailed reporting and usage data</li>
+    <li>Provide more detailed reporting and usage data for services</li>
+    <li>Provide reusable text message templates</li>
     <li>Explore ways to let services send one-off messages</li>
+    <li>Continue to explore other improvements to emails, text messages and letters<li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:52]: I still think this one is a bit vague – does this mean user research, prototyping, something else? Improvements to templates, functionality, pricing, content, something else?

Not a blocker if everyone is happy with this for now.

File: app/templates/views/roadmap.html [hunk 10]
```diff
@@ -15,83 +15,60 @@ <h1 class="heading-large">Roadmap</h1>
...
+    <li>Continue to improve our front-end code to meet our <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.accessibility_statement')}}">targets for accessibility</a></li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrate our applications from GOV.UK PaaS</a> to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:54]: This exactly duplicates something we’re currently working on.

Do we want to say ‘more applications’ or ‘Continue to migrate’ or something else to differentiate this.

Also, we should avoid having multiple links to the same destination on this page. Best to keep the first link to the blog post and Amazon ECS - then remove the rest.

File: app/templates/views/roadmap.html (hunk 9 above)
Comment (saimaghafoor) [reply app/templates/views/roadmap.html:52]: I played around with the original wording for a while (Continue to explore other innovations on the mobile and email communications market, and improvements to our product). Happy for any suggestions!

File: app/templates/views/roadmap.html [hunk 11]
```diff
@@ -15,83 +15,60 @@ <h1 class="heading-large">Roadmap</h1>
...
+    <li>Let services create letters in the Welsh language to comply with the Welsh Language Act 1993</li>
+    <li>Add attachments to letters</li>
+    <li>Improve our email functionality so users can add images, QR codes and have more formatting options</li>
+    <li>Make it easier for services to request to go-live</li>
+    <li>Make it easier for users to join a live service</li>
+    <li>Design and test reusable content patterns for text messages</li>
+    <li>Integrate with our letter provider's API to improve the letter sending process</li>
     <li>Improve our front-end code to meet our <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.accessibility_statement')}}">targets for accessibility</a></li>
-
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrate our database from GOV.UK PaaS</a> to one hosted using <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/rds/">Amazon RDS</a></li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:40]: We should avoid having multiple links to the same destination on this page. Best to keep the first instance of links to the blog post and Amazon ECS - then remove the rest.

File: app/templates/views/roadmap.html [hunk 12]
```diff
@@ -15,83 +15,60 @@ <h1 class="heading-large">Roadmap</h1>
...
-  <ul class="govuk-list govuk-list--bullet">
-
-    <li>ask recipients to enter their email address before they can download a file</li>
-    <li>choose the length of time that a file is available to download</li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2023/01/23/how-we-let-gov-uk-notify-users-set-their-own-branding/">Let services upload their own logo for email and letters</a><li>
+    <li>Let services choose their own two-factor authentication method</li>
+    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="https://www.notifications.service.gov.uk/using-notify/guidance/receive-text-messages/">receive text messages</a> without having to contact the Notify team first</li>
+    <li>Made it easier to find information and guidance on the Notify website</li>
+    <li>Made it quicker and easier to turn off <a class="govuk-link govuk-link--no-visited-state" href="https://www.notifications.service.gov.uk/features/trial-mode">trial mode restrictions</a></li>
+    <li>Developed first version of reusable content patterns for text messages</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:67]:
```suggestion
    <li>Designed a first draft of reusable content patterns for text messages</li>
```

File: app/templates/views/roadmap.html (hunk 9 above)
Comment (karlchillmaid) [reply app/templates/views/roadmap.html:52]: Maybe we don’t say anything?

I think it should be implicit/expected that we’re always working to improve our product.

If we can’t be specific about what we’re going to look at/do, then I’m not sure if this line adds much value.

Comment (saimaghafoor) [reply app/templates/views/roadmap.html:52]: Agree. Happy to remove.

File: app/templates/views/roadmap.html [hunk 13]
```diff
@@ -15,83 +15,58 @@ <h1 class="heading-large">Roadmap</h1>
...
-  <h3 class="heading-small" id="april-to-june">April 2023 onwards</h3>
+  <h3 class="heading-small" id="april-to-june">April to August 2023</h3>
 
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>Improve formatting options for letters</li>
-    <li>Provide more detailed reporting and usage data</li>
+    <li>Provide more detailed reporting and usage data for services</li>
+    <li>Provide reusable text message templates</li>
     <li>Explore ways to let services send one-off messages</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:51]:
```suggestion
    <li>Explore ways to make it easier for services to send one-off messages</li>
```

File: app/templates/views/roadmap.html [hunk 14]
```diff
@@ -15,83 +15,58 @@ <h1 class="heading-large">Roadmap</h1>
...
-
-  <p class="govuk-body">To protect files sent by email, service teams can now:</p>
-
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>ask recipients to enter their email address before they can download a file</li>
-    <li>choose the length of time that a file is available to download</li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2023/01/23/how-we-let-gov-uk-notify-users-set-their-own-branding/">Let services upload their own logo for email and letters</a><li>
+    <li>Let services choose their own two-factor authentication method</li>
+    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="https://www.notifications.service.gov.uk/using-notify/guidance/receive-text-messages/">receive text messages</a> without having to contact the Notify team first</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:63]:
```suggestion
    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="https://www.notifications.service.gov.uk/using-notify/guidance/receive-text-messages/">receive text messages</a> without having to contact the Notify team</li>
```

File: app/templates/views/roadmap.html [hunk 15]
```diff
@@ -15,83 +15,58 @@ <h1 class="heading-large">Roadmap</h1>
...
-
-  <h3 class="heading-small">Made sending files by email more secure</h3>
-
-  <p class="govuk-body">To protect files sent by email, service teams can now:</p>
-
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>ask recipients to enter their email address before they can download a file</li>
-    <li>choose the length of time that a file is available to download</li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2023/01/23/how-we-let-gov-uk-notify-users-set-their-own-branding/">Let services upload their own logo for email and letters</a><li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:61]:
```suggestion
    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2023/01/23/how-we-let-gov-uk-notify-users-set-their-own-branding/">upload their own logo for emails and letters</a><li>
```

File: app/templates/views/roadmap.html [hunk 16]
```diff
@@ -15,83 +15,58 @@ <h1 class="heading-large">Roadmap</h1>
...
-  <p class="govuk-body">To protect files sent by email, service teams can now:</p>
-
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>ask recipients to enter their email address before they can download a file</li>
-    <li>choose the length of time that a file is available to download</li>
+    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2023/01/23/how-we-let-gov-uk-notify-users-set-their-own-branding/">upload their own logo for emails and letters</a><li>
+    <li>Let services choose their own two-factor authentication method</li>
+    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="https://www.notifications.service.gov.uk/using-notify/guidance/receive-text-messages/">receive text messages</a> without having to contact the Notify team</li>
+    <li>Made it easier to find information and guidance on the Notify website</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:64]:
```suggestion
    <li>Updated the information and guidance on the Notify website</li>
```

File: app/templates/views/roadmap.html [hunk 17]
```diff
@@ -15,83 +15,58 @@ <h1 class="heading-large">Roadmap</h1>
...
-    <li>Let services choose their own two-factor authentication method</li>
-    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_receive_text_messages') }}">receive text messages</a> without having to contact the Notify team first</li>
-    <li>Make it easier to find information and guidance on the Notify website</li>
-    <li>Design reusable content patterns for text messages</li>
-    <li>Update the version of the <a class="govuk-link govuk-link--no-visited-state" href="https://design-system.service.gov.uk/">GOV.UK Design System</a> that we use</li>
+  
+    <li>Let services create letter templates that comply with the <a class="govuk-link govuk-link--no-visited-state" href="https://law.gov.wales/culture/welsh-language/welsh-language-act-1993">Welsh Language Act 1993</a></li>
+    <li>Add attachments to letters</li>
+    <li>Improve our email functionality so users can add images, QR codes and have more formatting options</li>
+    <li>Make it easier for services to request to go live</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:35]:
```suggestion
    <li>Make it easier to find information and guidance on the Notify website</li>
```

File: app/templates/views/roadmap.html [hunk 18]
```diff
@@ -15,83 +15,59 @@ <h1 class="heading-large">Roadmap</h1>
...
-    <li>Migrate one of our applications from GOV.UK PaaS to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
-    <li>Make it quicker and easier to turn off <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.trial_mode')}}">trial mode restrictions</a></li>
-    <li>Let services choose their own two-factor authentication method</li>
-    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_receive_text_messages') }}">receive text messages</a> without having to contact the Notify team first</li>
+  
+    <li>Let services create letter templates that comply with the <a class="govuk-link govuk-link--no-visited-state" href="https://law.gov.wales/culture/welsh-language/welsh-language-act-1993">Welsh Language Act 1993</a></li>
+    <li>Add attachments to letters</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:33]:
```suggestion
    <li>Let services attach extra pages, like forms, to letters</li>
```

File: app/templates/views/roadmap.html [hunk 19]
```diff
@@ -15,83 +15,59 @@ <h1 class="heading-large">Roadmap</h1>
...
-    <li>Migrate one of our applications from GOV.UK PaaS to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
-    <li>Make it quicker and easier to turn off <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.trial_mode')}}">trial mode restrictions</a></li>
-    <li>Let services choose their own two-factor authentication method</li>
-    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_receive_text_messages') }}">receive text messages</a> without having to contact the Notify team first</li>
+  
+    <li>Let services create letter templates that comply with the <a class="govuk-link govuk-link--no-visited-state" href="https://law.gov.wales/culture/welsh-language/welsh-language-act-1993">Welsh Language Act 1993</a></li>
+    <li>Let services attach extra pages, like forms, to letters</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:33]:
```suggestion
    <li>Let services attach pages, like forms, to letters</li>
```

**Review by karlchillmaid — APPROVED** [review]
I think this is ok from a content perspective now.

Could do with a fresh pair of eyes to check accuracy.

File: app/templates/views/roadmap.html [hunk 20]
```diff
@@ -15,83 +15,59 @@ <h1 class="heading-large">Roadmap</h1>
...
     <li>Make it easier to find information and guidance on the Notify website</li>
-    <li>Design reusable content patterns for text messages</li>
-    <li>Update the version of the <a class="govuk-link govuk-link--no-visited-state" href="https://design-system.service.gov.uk/">GOV.UK Design System</a> that we use</li>
+    <li>Make it easier for services to request to go live</li>
+    <li>Make it easier for users to join a live service</li>
+    <li>Design and test reusable content patterns for text messages</li>
+    <li>Integrate with our letter provider’s API to improve the letter sending process</li>
     <li>Improve our front-end code to meet our <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.accessibility_statement')}}">targets for accessibility</a></li>
-
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrate our database from GOV.UK PaaS</a> to one hosted using <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/rds/">Amazon RDS</a></li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:41]:
```suggestion
    <li>Migrate our database from <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">GOV.UK PaaS</a> to one hosted using <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/rds/">Amazon RDS</a></li>
```

**Review by leohemsted — APPROVED** [review]
